prime-router/settings/staging/0169-epic-staging-sender-receiver.yml

```diff
@@ -0,0 +1,112 @@
+# To submit this to localhost, run:
+# ./prime multiple-settings set --env staging --input ./settings/staging/0169-epic-staging-sender-receiver.yml
+#
+# Description:
+#   These following setting are for Epic Systems Integration Testing
+#   Order sender (epic.etor-nbs-orders)
+#   Result Receiver (epic.etor-nbs-results)
+#
+---
+- name: "epic"
+  description: "Epic Systems Integration Testing"
+  jurisdiction: "STATE"
+  stateCode: LA
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders:
+    - !<full-elr>
+      name: "etor-nbs-orders"
+      organizationName: "epic"
+      format: "HL7"
+      customerStatus: "active"
+      schemaName: ""
+      processingType: "async"
+      allowDuplicates: true
+      senderType: null
+      primarySubmissionMethod: null
+      topic: "etor-ti"
+      version: null
+      createdBy: null
+      createdAt: null
+  receivers:
+    - name: "etor-nbs-results"
+      organizationName: "epic"
+      topic: "etor-ti"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "metadata/hl7_mapping/ORU_R01/ORU_R01-base"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "EPIC"
+        receivingApplicationOID: null
+        receivingFacilityName: "LA"
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: {}
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "standard"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01')"
+        - "(%performerState.exists() and %performerState = 'LA') or (%patientState.exists() and %patientState = 'LA')"
```

Comment:
This routing filter will change once we get clarity on what the id will be for the receiver. The filter will be something like this: `Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(value = 'receiver-id').exists()` where we'll replace `receiver-id`